apps/design-land/src/app/core/code-preview/component/code-preview.component.scss

```diff
@@ -0,0 +1,11 @@
+:host {
+	border: 1px solid gray;
```

Comment:
color needs to use the daffodil palette variables and separated into code preview's own theme file.